package-lock.json

```diff
@@ -19199,9 +19199,9 @@
       "dev": true
     },
     "prettier": {
-      "version": "1.19.1",
-      "resolved": "https://registry.npmjs.org/wp-prettier/-/wp-prettier-1.19.1.tgz",
-      "integrity": "sha512-mqAC2r1NDmRjG+z3KCJ/i61tycKlmADIjxnDhQab+KBxSAGbF/W7/zwB2guy/ypIeKrrftNsIYkNZZQKf3vJcg==",
+      "version": "npm:prettier@1.19.1",
+      "resolved": "https://registry.npmjs.org/prettier/-/prettier-1.19.1.tgz",
+      "integrity": "sha512-s7PoyDv/II1ObgQunCbB9PdLmUcBZcnWOcxDh7O0N/UwDEsHyqkW+Qh28jW+mVuCdx7gLB0BotYI1Y6uI9iyew==",
```

Comment:
@swissspidy 👋 not sure this is incorrect, the version isn't aliased so shold be resolved as `pretter` from npm. There's another entry in your lockfile where the version is resolved to `npm:wp-prettier@1.19.1`.

This update will have been done by npm when doing `npm install` (called directly by Dependabot). Dependabot can't currently update aliased packages in your `package.json` but I believe this is done as a side-effect of trying running `npm install` on the updated lockfile.

Reply:
Thanks for looking into this!

Not sure I am following you 100% though. 

So far I wasn't able to replicate this behavior by running `npm install`, i.e. removing the lock file and running `npm install` results in zero changes.

Would you say I should just wait for the PR you have been working on, or do something else to fix this on our end?